packit_service/worker/helpers/build/build_helper.py

```diff
@@ -385,14 +385,24 @@ def get_check_cls(
         chroot: str = None,
         project_event_identifier: Optional[str] = None,
         identifier: Optional[str] = None,
+        package: Optional[str] = None,
+        template: Optional[str] = None,
     ):
+        if project_event_identifier:
+            project_event_identifier = project_event_identifier.replace(":", "-")
+
+        if template is not None:
+            return template.format(
```

Comment:
should we wrap this to `try-except` for the case of misconfigured template?

Reply:
that's a good idea :+1: